gl-zkPass/zkpass-sdk
# Update Rust demo and dependencies

Asana: https://app.asana.com/0/1206389362636911/1207189648450385/f

Changes:
* Update Rust Demo to latest query improvement
* Update test data set

How to test:
Note: test this PR in your local until further notice
1. Go to rust directory
2. build with `cargo build -r`
3. add `export LD_LIBRARY_PATH=./lib`
4. Run the demo with `./target/release/zkpass-demo r0 test/data/dewi-profile.json test/data/bca-insurance-dewi-dvr.json`
5. Expected result
```bash
...
#### verification completed []
json-result={
  "name": "Dewi",
  "result": true
}
>> output list:
key=name, value=Str("Dewi")
key=result, value=Bool(true)
<< end of list
the query result is true
```
